def fun1():
	return "fail"

print fun1()
